external/segger_rtt/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+FetchContent_Declare(
+    segger_rtt
+    GIT_REPOSITORY https://github.com/SEGGERMicro/RTT
+    GIT_TAG ff57c3d516ea59b0a2d4d5cafb33a48572b09c0b # v7.54
+    CONFIGURE_COMMAND ""
```

Comment:
Do these empty commands do anything?

Reply:
Yes, they make sure CMake doesn't try to configure or build this dependency. Since there is no CMake support in the SEGGERMicro/RTT project.

So we use FetchContent here to clone the repository more or less.